clang/unittests/Format/TokenAnnotatorTest.cpp

```diff
@@ -1591,6 +1591,12 @@ TEST_F(TokenAnnotatorTest, UnderstandsLambdas) {
   EXPECT_TOKEN(Tokens[15], tok::arrow, TT_TrailingReturnArrow);
   EXPECT_TOKEN(Tokens[17], tok::l_brace, TT_LambdaLBrace);
 
+  Tokens = annotate("auto l = [] -> struct S { return {}; };");
```

Comment:
Is the assignment to a variable necessary? None of the other lambda tests have this

Reply:
Not really. I removed the assignment.